docs/understanding-the-flutterflow-ui/Projects/how-to-collaborate-on-projects.md

```diff
@@ -0,0 +1,163 @@
+---
+slug: how-to-collaborate-on-projects
+title: How to Collaborate on Projects
+tags: []
+toc_max_heading_level: 5
+sidebar_position: 4
+
+---
+FlutterFlow allows for multiple ways to collaborate on a project as well as the ability to manage the collaboration through permissions.
+
+## Sharing a Project with a User
+
+You can also invite users to your project who are not part of your organization. For instance, you might want to share your work with clients, stakeholders, or team members of the client.
+
+When you invite users to your project, you have the flexibility to assign them different roles depending on their level of involvement. For example, you can grant editing privileges to collaborators who are actively working on the project while assigning read-only access who need to view the progress. Alternatively, you can also make them the owner of the project, giving them full control.
+:::info
+- When you add a user, the default role assigned is the *Editor* role.
+- Users with read-only access will only be able to access that specific project and won't be able to access any shared *Teams* libraries (e.g., custom code, design system).
+- Read-only users added to *Teams*/*Organization* will not count against your *Teams* seat count.
+- You must verify your email before inviting users.
+  :::
+
+### How to Invite Users
+
+<div style={{
+    position: 'relative',
+    paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
+    height: 0,
+    width: '100%'
+}}>
+    <iframe 
+        src="https://demo.arcade.software/9XkpLuC3tpiFFapWi7ao?embed&show_copy_link=true"
```

Comment:
Arcade has a feature where you can blur some information in the screenshot. I think we should blur the email IDs. This applies to all the emails in this page.

Reply:
What's it to you if I wanna dox myself?

fixed.